services/app-api/action/changeStatusAny.js

```diff
@@ -43,11 +41,13 @@ export const changeStatusAny = async (event, config) => {
   }
 
   try {
-    await Promise.all(
-      [CMSWithdrawalNotice, stateWithdrawalReceipt]
-        .map((f) => f(updatedPackageData, config))
-        .map(sendEmail)
+    const theEmails = await Promise.all(
+      config.emailFunctions.map(
```

Comment:
Did we define emailFunctions on other events that call changeStatusAny? I only see it on defaultWithdrawConfig?

Reply:
looks like we only use this for withdraws so far......so all is good!